Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,329 @@
+
+***
+
+## Versions & Differences: 
+
+#### Tested:
+
+* copy-X light[^84759][^107929][^81628][^Kenny][^101786]
+* copy-X professional[^108150][^48393][^82475][^DE_EM2DX][^Petterson]
+
+[^84759]: [Black Cats and Pointed Hats (USA) (En,De)](http://redump.org/disc/84759/)
+[^107929]: [Robinson Crusoe (USA)](http://redump.org/disc/107929/)
+[^81628]: [Webmaster (USA))](http://redump.org/disc/81628/)
+[^108150]: [Case for TKKG, A - The Mystery of the Mayan Treasure (USA) (En,De)](http://redump.org/disc/108150/)
+[^48393]: [Emergency 4 - Global Fighters for Life (Germany) (Disc 1)](http://redump.org/disc/48393/)
+[^82475]: [Sluzhba 911 (Russia) - Remnants](http://redump.org/disc/82475/)
+[^DE_EM2DX]: [German Emergency 2 Deluxe](https://archive.org/details/DRML-copy-X-samples)
+[^Petterson]: [Petterson und Findus - Hühnertanz und Katerzirkus](https://archive.org/details/DRML-copy-X-samples)
+[^Kenny]: [Kenny's Adventure](https://archive.org/details/DRML-copy-X-samples)
+[^101786]: [Max and the Magician (Europe) (En,Fr,De,Es)](http://redump.org/disc/101786/)
+
+##### copy-X light:
+
+This behavior has been tested, but it is still unconfirmed if it specifically corresponds to copy-X light. 
+Basic ring protection via dummy files intersected[^Dummy_Confirmation] by rings. No software protection is used unless otherwise supplied by the game itself[^84759]. Can be circumvented by simply not copying the dummy files when copying the rest of the game from the disc.
+
+[^Dummy_Confirmation]: Confirmed via checking dummy files in a hex editor. These dummy files are always intersected by rings (0x55 on redump-images, varies on others depending on what the dumping program does with those errors).
+
+##### copy-X professional:
+
+This behavior has been tested, but it is still unconfirmed if it specifically corresponds to copy-X professional.
+Builds upon copy-X light. Contains everything described there, but also adds software protection in the form of a disc check, which appears to additionally check one of the dummy files. Copy-X professional also checks if the kernel debugging software SoftICE is active via a process called SoftICE-Test, checking both for "SoftICE for Windows 95" and for "SoftICE for Windows NT", ending game execution if detected when booting the game. This seems to be the only debugger protected against, likely due to the popularity of this debugger at the time.
+
+#### Untested:
+
+* copy-X audio
+* copy-X trial maker
+
+##### copy-X audio:
+
+No samples have been tested at the moment, so exact specifics cannot be confirmed. Used to protect Audio CDs.
+
+##### copy-X trial maker:
+
+No samples currently known, and thus no information either.
+
+***
+
+## Samples:
+ 
+#### Confirmed:
+
+* copy-X light
+	* Black Cats and Pointed Hats (USA) (En,De) [^84759]
```

Comment:
Nit: Please remove the extra space between the footnote and the languages.

Reply:
Fixed.